src/components/card/card.scss

```diff
@@ -69,6 +69,13 @@
     font-medium;
 }
 
+:host([selectable]) {
+  .header {
+    // prevents overlap with checkbox
+    padding-inline-end: calc(theme("spacing.3") + theme("spacing.5"));
```

Comment:
Nitpick : can we replace `calc` with `theme("spacing[8.5]")`

Reply:
Yeah, I think I'll replace that with a comment to explain that it's the heading + no-overlap spacing.